### PR TITLE
feat: refactor documentation website layout

### PR DESCRIPTION
**Pull request by iamnamananand996**
… (#590)

Because

- refactor documentation website layout

This commit

- refactor documentation website layout

### CODE REVIEW

File: content.config.tsx [hunk 1]
```diff
@@ -304,25 +362,152 @@ export function getNavbar(appType: InstillAppType, isDark: boolean): NavConfig {
...
+            label: "Resources",
+          },
+          {
+            key: "docs-nav-dropdown-product-website",
+            href: "/",
+            className: "navbar-dropdown-menu",
+            position: "right",
+            label: "Product Website",
+            iconElement: (
+              <Icons.ArrowUpRight className="h-4 w-5 stroke-instillGrey95 hover:stroke-instillBlue50 dark:stroke-instillGrey15 dark:hover:stroke-instillBlue50" />
```
Comment (EiffelFly) [inline content.config.tsx:387]: All of these icon should be vertical center

**Review by EiffelFly — COMMENTED** [review]
Everything looks great, except some center issue. And there is a visual bug 

https://github.com/instill-ai/instill.tech/assets/57251712/41504897-e87e-4617-a39e-49e40140a36e

Please fix it

File: content.config.tsx (hunk 1 above)
Comment (iamnamananand996) [reply content.config.tsx:387]: okay

Comment (iamnamananand996) [reply content.config.tsx]: updated suggested changes

**Comment by EiffelFly** [comment]
@iamnamananand996 When click the instill icon in the Navbar, it should redirect user back to the product website

**Comment by iamnamananand996** [comment]
> @iamnamananand996 When click the instill icon in the Navbar, it should redirect user back to the product website. I think we miss that in the previous version, Please help me add it up~

updated now 
cc - @EiffelFly

**Pull request merged**